Editor/Inspector/Menu/MenuItemGUI.cs

```diff
@@ -215,11 +220,46 @@ public void DoGUI()
             EditorGUILayout.BeginHorizontal();
 
             EditorGUILayout.BeginVertical();
-            EditorGUI.BeginChangeCheck();
-            EditorGUILayout.PropertyField(_name, G("menuitem.prop.name"));
-            if (EditorGUI.EndChangeCheck())
+
+            EditorGUILayout.BeginHorizontal();
+            var needsRichLabel = (!string.IsNullOrEmpty(_prop_label.stringValue) || _isTryingRichLabel);
+            if (!needsRichLabel)
+            {
+                EditorGUI.BeginChangeCheck();
+                EditorGUILayout.PropertyField(_name, G("menuitem.prop.name"));
```

Comment:
If we're linked, we should just show the name property (which can link directly to the GameObject if this is a `ModularAvatarMenuItem`). If we're not linked, we should show the label field always (and conditionally show the rich text preview).